docs/content/en/install.md

```diff
@@ -7,7 +7,7 @@ position: 102
 ### Precompiled binaries
 
 Precompiled `chproxy` binaries are available [here](https://github.com/ContentSquare/chproxy/releases).
-Just download the latest stable binary, unpack and run it with the desired [config](#configuration):
+Just download the latest stable binary, unpack and run it with the desired [config](/configuration/default):
```

Comment:
This fixes the broken link but I'm unsure whether this is intended target page, as it is the last of its `category` on the website.

Reply:
I think that's the correct link; other links explain each individual parts of the config; this one links to the default config